src/modules/migration/MigrationListItem.tsx

```diff
@@ -53,18 +53,17 @@ export const MigrationListItem = ({
   v3Rates,
   userControlledCollateral,
   canBeEnforced,
+  isSupplyList,
 }: MigrationListItemProps) => {
   const theme = useTheme();
   const { currentMarket, currentMarketData } = useRootStore();
-  const isDesktop = useMediaQuery(theme.breakpoints.up('xl'));
-  const isTablet = useMediaQuery(theme.breakpoints.up(655));
-  const isMobile = useMediaQuery(theme.breakpoints.down(655));
+  const isMobile = useMediaQuery(theme.breakpoints.down(1125));
 
-  const assetColumnWidth =
-    isMobile && !isTablet ? 45 : isTablet && !isDesktop ? 80 : isDesktop ? 120 : 80;
   const baseColor = disabled === undefined ? 'text.primary' : 'text.muted';
   const baseColorSecondary = disabled === undefined ? 'text.secondary' : 'text.muted';
 
+  const loadingRates = v3Rates?.ltv === undefined && v3Rates?.liquidationThreshold === undefined;
```

Comment:
This would be a lot easier to handle if we had two different components, but for now the check assumes the data is still loading if both are undefined.

Reply:
We have the same problem in the dashboard/markets, we use the same for both of them. I second doing this change in the future before it gets hard to add something.